services/QuillLMS/app/models/concerns/translatable.rb

```diff
@@ -122,4 +127,11 @@ def open_ai_prompt(locale:)
   private def translatable_text
     data[default_translatable_field]
   end
+
+  private def clean_deprecated_translations(translatable_text:, field_name:)
+    translation_mappings.joins(:english_text)
+      .where(field_name:)
+      .where.not(english_text: { text: translatable_text })
+      .destroy_all
```

Comment:
Take or leave, it is a little surprising to see methods called `create_` and `create_` and then see a `destroy` side effect. I wonder if it would be better to move this up in the chain.